.github/workflows/reviewdog.yml

```diff
@@ -0,0 +1,30 @@
+name: reviewdog
+on:
+  pull_request:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref }}
+  cancel-in-progress: true
+
+jobs:
+  reviewdog:
+    name: 🐶 Reviewdog
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: reviewdog/action-setup@v1
+        with:
+          reviewdog_version: latest
+
+      - uses: actions/setup-python@v3
+        with:
+          python-version: "3.10"
```

Comment:
using oldest runtime might be better

Reply:
Why? 🤔

Reply:
so as to check if we are fine with old way of doing things

Reply:
similar to your change in php (7.3 instead of 7.4)

Reply:
it's tooling, we can adjust later.